.github/workflows/federated-integ-test.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+name: Federated integ test
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      - v1.*
+  pull_request:
```

Comment:
we don't want heavy integration tests to be triggered on pull_request.
You should use the workflow_dispatch trigger to execute on demand on PR.

Reply:
yep, this is a test right now. I will probably remove the whole file once i get this running (if i get it)

Will create a separate pr for that

Reply:
Hey @quentinDERORY  I removed the github action file form this PR